app/Modules/Group/Events/GroupStatusUpdated.php

```diff
@@ -23,6 +23,11 @@ class GroupStatusUpdated extends GroupEvent
      */
     public function __construct(public Group $group, public GroupStatus $newStatus, public GroupStatus $oldStatus)
     {
+
```

Comment:
1. Comment is out of date with what code represents.
2. Indentation does not match surrounding code.
3. As we had discussed a few meetings ago, it is not good coding practice to use these integers as "magic numbers". For instance, `2` here represents `config('groups.statuses.active.id')`, which is more verbose but provides much more semantic information for someone reading this code (arguably even enough to obviate the comment).
4. Since what we are really doing is sending for anything that is *not* in "applying" status, it would be even cleaner to write this as: `if ($this->group->group_status_id == config('groups.statuses.applying.id') return;`

Reply:
We didn't discuss magic numbers.  It was a comment that you made in code.  Personally I don't see an issue with it, but will follow your direction.

Reply:
My comments describe above about code formatting/indentation/magic numbers are resolved, but I still don't think this code, having an early return from the constructor, has any meaningful functionality. As such, keeping it in may just cause confusion.

Reply:
Resolving since these comments are addressed in later commit.